lib/rules/cli/4.1.yaml

```diff
@@ -186,6 +186,7 @@
     :restart: --restart=<value>
     :save_config: --save-config=<value>
     :schedule: --schedule=<value>
+    :sleep:  -- sleep <value>
```

Comment:
@kasturinarra should it be `--sleep=<value>`

Reply:
It should not have '=',  the command which i wanted to run is "oc create cronjob <name> <imagename> --schedule='* * * * *' -- sleep 180

Reply:
@kasturinarra if `:sleep` is not part of the `oc command` then this shouldn't be part of the yaml file.  We only add options that are specific to the `oc` client command